### DISCUSSION TITLE
Develop a training example for detection tasks 基于 MMEngine 实现检测任务的训练用例

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#304] [MMEngine]
# Develop a training example for detection tasks 
 基于 MMEngine 实现检测任务的训练用例 

To implement a training pipeline for detection tasks using MMEngine,please refer to the following link for detailed instructions: [[https://github.com/open-mmlab/mmengine/issues/1117](https://github.com/open-mmlab/mmengine/issues/1117)](https://github.com/open-mmlab/mmengine/issues/1117](https://github.com/open-mmlab/mmengine/issues/1117)) 

基于 MMEngine 实现检测任务的训练用例
 
 具体请见：[[https://github.com/open-mmlab/mmengine/issues/1117](https://github.com/open-mmlab/mmengine/issues/1117)](https://github.com/open-mmlab/mmengine/issues/1117](https://github.com/open-mmlab/mmengine/issues/1117)) 
 

## Task Information

**Repository**：MMEngine

**Difficulty**：Medium

**Points**：30

**Estimated Time Consumption**：14 days

## Task Status

**Claimant's Github ID**：None

**Claim Time**：2023年08月25日

**PR Link**：https://github.com/open-mmlab/mmengine/pull/1134

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by Shengshenlan** [comment]
Claim the mission

**Comment by crazysteeaam** [comment]
This task has been done at https://github.com/open-mmlab/mmengine/pull/1134.
